doit/cmd_run.py

```diff
@@ -168,31 +183,42 @@ def _execute(self, outfile,
             # user defined class
             reporter_cls = reporter
 
+        # check_file_uptodate
+        if isinstance(check_file_uptodate, six.string_types):
+            if check_file_uptodate not in CHECKERS:
+                msg = ("No check_file_uptodate named '%s'."
+                       " Type 'doit help run' to see a list "
+                       "of available checkers.")
+                raise InvalidCommand(msg % check_file_uptodate)
```

Comment:
I know this is copy and paste from another part of the code but when adding new code please prefer the use of `{}`  and `format`